coucharchive

```diff
@@ -522,6 +580,11 @@ def main():
     sub['create'].add_argument(
         '-o', '--output', dest='output', action='store', required=True,
         help='path to archive to create')
+    sub['create'].add_argument(
+        '--no-queue-control', dest='no_queue_control',
+        action='store_true', default=False,
+        help='deactivate throttle request if origin couchdb `run_queue` '
```

Comment:
Sorry for the double modification (following @bravier's remark), but *deactivate → disable* would be more standard I think!

Other proposals:
- throttle request → throttling of requests
- ~origin~  *(I think now it's both source and target)*
- couchdb → CouchDB
- `run_queue` system info is too high → run queue is too long

Reply:
Thanks!
It give:
```py
        help='disable throttling of requests if CouchDB `run_queue` is too '
             'high')
```